materialize/provider.go

```diff
@@ -10,8 +10,8 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	_ "github.com/jackc/pgx/stdlib"
 	"github.com/jmoiron/sqlx"
-	_ "github.com/lib/pq" //PostgreSQL db
```

Comment:
I think this is getting pulled in somewhere else too, because it's not showing up as removed in go.mod.

Reply:
Looks like it is being used for `pq.QuoteLiteral` in the secrets. Looks like this was the suggested solution in https://github.com/MaterializeInc/terraform-provider-materialize/pull/18. Is there a better alternative that doesn't use pg?

Reply:
Yes! I linked the equivalent pgx functions here: https://github.com/MaterializeInc/terraform-provider-materialize/issues/27#issue-1590190068